packages/web/app/components/PatientInfoPane/PatientInfoPane.jsx

```diff
@@ -153,6 +153,7 @@ const ProgramRegistryDisplay = memo(({ patient, readonly }) => (
         fallback="Program registry"
       />
     }
+    id={PANE_SECTION_IDS.PROGRAM_REGISTRY}
```

Comment:
This is an existing issue. The id here is used in the query key and then when we clear the query cache these queries don't reset.